mysql/Oracle/README.md

```diff
@@ -0,0 +1,25 @@
+# Oracle's MySQL grammar
+
+##General
+Since more than a decade the MySQL GUI dev tools team at Oracle offers the open source [MySQL Workbench](https://github.com/mysql/mysql-workbench) product, which uses ANTLR4 for all MySQL code parsing tasks. This requires to translate all changes from the [MySQL server grammar](https://github.com/mysql/mysql-server/blob/8.0/sql/sql_yacc.yy) to ANTLR4, which is an ongoing effort to always stay up-to-date with the lastest and greatest server features. The current grammar supports the server versions 5.6, 5.7 and 8.0.
```

Comment:
There is a good practice of breaking long text lines into several short ones (up to 80-120 chars). It's better for comparison (diff) of different versions and more convenient for review to point certain line, not the whole paragraph.

Reply:
I'm not sure I follow that argumentation about better comparison, but I added linebreaks anyway. They don't change how the file is rendered in HTML, but increase the plain text readability.